Rubberduck.Inspections/Concrete/StepIsNotSpecifiedInspection.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Rubberduck.Inspections.Abstract;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Parsing.Inspections.Resources;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Parsing.Grammar;
+using Antlr4.Runtime.Misc;
+using Antlr4.Runtime;
+using Rubberduck.Parsing;
+using Rubberduck.VBEditor;
+using Rubberduck.Inspections.Results;
+using static Rubberduck.Parsing.Grammar.VBAParser;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    public sealed class StepIsNotSpecifiedInspection : ParseTreeInspectionBase
+    {
+        public StepIsNotSpecifiedInspection(RubberduckParserState state) : base(state, CodeInspectionSeverity.Warning) { }
+
+        public override Type Type => typeof(StepIsNotSpecifiedInspection);
```

Comment:
nothing to do with your PR, but we really wanted to get rid of this...

Reply:
@Vogel612 It's deprecated already... and actually needs to be removed in order to build 😄